go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/formancehq/payments
 
-go 1.23.2
+go 1.24.4
```

Comment:
that would hvae been better in another PR I think in case we need to revert, but better to have it that not